### ISSUE TITLE
Are you guys closing down?

### ISSUE THREAD

**Issue by sergiogranadamoreno** [enzienaudio/heavy#266]
Just a few minutes ago I lost access with Heavy (https://enzienaudio.com/) and then found this post on twitter:
![img-20180731-wa0000](https://user-images.githubusercontent.com/4422632/43488864-a8dae47e-94e0-11e8-88a3-5569097d6b9d.jpg)

Is this true??

**Comment by diplojocus** [comment]
Yes unfortunately it's true, though service should be up and running as usual for the next month.

**Comment by dbaxaudio** [comment]
Sad times :(

I really enjoyed using Heavy with Unity experimentation. Hadn’t did anything in a while but was planning on doing lots of stuff in the future with it.

Joe you were always really helpful with support, thanks so much for all your time and assistance with my stupid queries!

An thanks so much for an amazing product/service Joe, Martin, Rachel and Chris! Best of luck with whatever you are all moving on to. 

David

**Comment by moshang** [comment]
Also terribly sad to see you go ;^( Selfishly more so because I'm in the middle of a project using Heavy. I've managed to build the c/c++ code locally as per the instructions in the hvcc folder, but I'm stuck converting the resulting files to the Unity plugins that I need for the project. I see the  -g (generators) build switch is marked as todo. Is this the missing link in the puzzle for generating the platform specific plugins? If so, is there a timeline you're willing to share for having that feature implemented? Thanks so much for your awesome work on Heavy!

**Comment by diplojocus** [comment]
@moshang add `-g unity` to the command and it'll generate a unity folder with xcode/vs2015/linux makefile projects to build the plugin.

**Comment by moshang** [comment]
Thanks a lot - I'll give that a go.
